java/ql/src/Language Abuse/EmptyMethod.md

```diff
@@ -0,0 +1,36 @@
+## Overview
+
+An empty method may indicate that an implementation was intended to be provided but was accidentally omitted. When using the method, it will not be clear that it does not provide an implementation and with dynamic dispatch, resolving to a blank method may result in unexpected program behaviour.
```

Comment:
US spelling
```suggestion
An empty method may indicate that an implementation was intended to be provided but was accidentally omitted. When using the method, it will not be clear that it does not provide an implementation and with dynamic dispatch, resolving to a blank method may result in unexpected program behavior.
```